test/powershell/engine/Formatting/ErrorView.Tests.ps1

```diff
@@ -211,5 +211,14 @@ Describe 'Tests for $ErrorView' -Tag CI {
 
             $e | Should -BeExactly (Get-Error | Out-String).Trim([Environment]::NewLine.ToCharArray())
         }
+
+        It 'Renders the passed error instead of the last one' {
+            # PR #25627
```

Comment:
```suggestion
```

Reply:
We never add such comments.

Reply:
The point is that if I encountered this test without the comment, I'd be very confused why should this scenario be tested at all.

Reply:
Because the name of the test sufficiently describes the scenario. If not, then come up with a better title.

Reply:
Sigh, whatever, removed.